shakenfist/client/upgrade.py

```diff
@@ -55,7 +55,9 @@ def main():
     print()
 
     min_version = None
-    if 'unknown' in versions:
+    if len(versions) == 0:
```

Comment:
[PEP8 Nazi] To be pythonic this should be `if not versions:` [/PEP8 Nazi]
I'm surprised the linter did not have a whinge.